app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def searchable?
 
   def auto_complete_search(name, val, options = {})
     path = options[:path] || send("#{controller_name}_path")
```

Comment:
> Or, what about just adding the auto_complete_search to the path on pages where it's needed?

That would be almost every single index page, I think, seems overkill.  This default is much better for most of the app, it's just in the filter page and one or two other places that we override it.

I wonder about changing the parameter so you can differentiate between :path (a prefix) which needs auto_complete_$NAME appending, or another which is a full path.